target_chains/cosmwasm/contracts/pyth/Cargo.toml

```diff
@@ -40,6 +40,7 @@ cosmwasm-schema = "1.1.9"
 osmosis-std = "0.15.2"
 pyth-sdk-cw = { path = "../../sdk/rust" }
 pythnet-sdk = { path = "../../../../pythnet/pythnet_sdk" }
+wormhole-cosmwasm =  {git = "https://github.com/wormhole-foundation/wormhole"}
```

Comment:
can you please lock it to a commit?
I think wormhole folks should really consider releasing their packages.